geoalchemy2/elements.py

```diff
@@ -113,6 +113,35 @@ def desc(self):
         return binascii.hexlify(self.data)
 
 
+class EWKBElement(_SpatialElement, functions.Function):
+    """
+    Instances of this class wrap a EWKB value. Geometry values read
+    from the database are converted to instances of this type. You may
+    need to create instances of ``EWKBElement`` yourself.
+
+    Note: you cannot create ``EWKBElement`` objects directly from Shapely
```

Comment:
I would not mention Shapely here. The `shape` module is the only place where Shapely should be mentioned in my opinion. Remember that using Shapely with GeoAlchemy is totally optional.